lighthouse-cli/index.js

```diff
@@ -6,7 +6,12 @@
  */
 'use strict';
 
-require('./bin.js').begin().catch(err => {
+require('./bin.js').begin().then((lhr) => {
+  // Exit with non-zero exit code if LHR contains runtime error.
+  if (lhr && lhr.lhr.runtimeError) {
```

Comment:
ideally we can handle this in `run.js` with the other runtime error/exiting code